components/common/layout/Footer.js

```diff
@@ -99,7 +99,7 @@ const Footer = () => (
               <ExternalLink href={CROWDFUNDING_CAMPAIGN.options.link}>
                 <Text id="footer.wir-on" />
                 <span>&nbsp;&nbsp;</span>
-                <IconImage name="molamola" color="#1a9582" />
+                <MolamolaIcon color="#1a9582" />
```

Comment:
you pass color, but doesn't use it in `MolamolaIcon` component

Reply:
Fixed.